policy_definitions/deploy_log_analytics_agent/policy_assignment_qby_deploy_log_analytics_agent.tmpl.json

```diff
@@ -0,0 +1,28 @@
+{
+    "type": "Microsoft.Authorization/policyAssignments",
+    "apiVersion": "2022-06-01",
+    "name": "QBY-Deploy-LAW-Agent",
+    "location": "${default_location}",
+    "dependsOn": [],
+    "identity": {
+        "type": "SystemAssigned"
+    },
+    "properties": {
+        "description": "Deploy Log Analytics extension for Windows virtual machines if the virtual machine image is in the list defined and the extension is not installed.",
+        "displayName": "Deploy - Configure Log Analytics extension to be enabled on Windows virtual machines",
+        "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/Deploy-Log-Analytics-Agent-to-Windows-VM",
+        "enforcementMode": "Default",
+        "nonComplianceMessages": [
+            {
+                "message": "Log Analytics extension is not configured for VM."
+            }
+        ],
+        "listOfImageSKUs": {
+            "value": {
+                "value": "2022-datacenter-azure-edition"
+            }
```

Comment:
can we overwrite this at the customer to avoid changing the archetype lib, to add a new SKU?